templates/docs_libs_placeholder.html

```diff
@@ -2,6 +2,7 @@
 {% load static %}
 
 {% block extra_head %}
+  <link href="{% static 'css/styles.css' %}" rel="stylesheet" />
```

Comment:
@rbbeeston does the avatar partial require tailwind?

Reply:
The avatar partials currently use tailwind, yes. We can convert it as another ticket.